parm/archive/gefs_atmos.yaml.j2

```diff
@@ -0,0 +1,48 @@
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+{% set head = RUN + ".t" + cycle_HH + "z." %}
+
+gefs_atmos:
+    name: "GEFS_ATMOS"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gefs_atmos.tar"
+    required:
+#select mem%03d and ensstat files required
+{% set file_set = [] %}
+{% set membr = ["ensstat"] %}
+{% for mem_nm in range(0, NMEM_ENS + 1) %}
+    {% do membr.append("mem" ~ '%03d' % mem_nm ) %}
+{% endfor %}
+
+{% for mem in membr %}
+    {% for res in ['0p25', '0p50', '1p00'] %}
+        {% set tmpl_dict = ({ '${ROTDIR}':ROTDIR,
+                              '${RUN}':RUN,
+                              '${YMD}':cycle_YMD,
+                              '${HH}':cycle_HH,
+                              '${GRID}':res,
+                              '${MEMDIR}':mem }) %}
+
+        {% set COMIN_ATMOS_GRIB = COM_ATMOS_GRIB_GRID_TMPL | replace_tmpl(tmpl_dict) %}
+
+        # Select pgrb and grib files to copy to the atardir
+        {% if RUN == "gefs" %}
+            {% if path_exists(COMIN_ATMOS_GRIB) %}
+                {% for fhr in range(FHMIN_GFS, FHMAX_GFS + FHOUT_GFS, FHOUT_GFS) %}
```

Comment:
1. We'll need to find a way to incorporate two forecast frequencies here. The first frequency is `FHOUT_HF_GFS` and the second frequency is `FHOUT_GFS`. The maximum lead hour for `FHOUT_HF_GFS` is `FHMAX_HF_GFS`.  These variables are already defined in `config.base`, but `FHOUT_HF_GFS` and `FHMAX_HF_GFS` may need to be added to `keys` in `scripts/exglobal_archive.py`. 
2. For replay cases, `FHMIN_GFS` should be replaced by `OFFSET_START_HOUR`. Some logic needs to be added here stating if `REPLAY_ICS` is `YES`, then `OFFSET_START_HOUR` should be used instead of `FHMIN_GFS` as the minimum forecast hour. `REPLAY_ICS` and `OFFSET_START_HOUR` may need to be added to  `keys` in `scripts/exglobal_archive.py`.